### ISSUE TITLE
b-table "@mouseenter" - access DOM element?

### ISSUE THREAD

**Issue by morph3us-net** [buefy/buefy#3235]
I am trying to add some features to filegator, a self hosted file storage. [[github repo]](https://github.com/filegator/filegator) [[demo]](https://demo.filegator.io/#/)

I created a new vue component showing a file icon depending on the current row's file's extension.
I added it to the same column as a dropdown menu, which is represented as 3 dots (for mobile devices).
I would like to initially hide the three dots and only display them if the user hovers over the corresponding row. (And hide the file icon.)
For this I would like to dynamically add an "is-hidden" helper class to either one of the two divs. (So I can override this behavior for mobile devices.)

![buefy-table-filegator](https://user-images.githubusercontent.com/43741267/111386621-cacf1d00-86ac-11eb-83ce-d52273f25dd9.jpg)

I am console logging the @mouseover event and it looks like it only provides the actual content of the row, no reference to objects or the DOM element. Do I miss something? Could someone tell me if its even possible to do something like that?

I set up a [codesandbox](https://codesandbox.io/s/buefytablehover-4v86r?file=/src/App.vue).
Basically I would like to switch between "div-red" and "div-blue" while hovering over the corresponding row.

Sorry if its inappropriate to ask something like this here.

**Comment by service-paradis** [comment]
Thanks for the request.
I agree, we should also send the original event back. I will submit a PR for this soon :)

**Comment by morph3us-net** [comment]
Wow! Thank you so much for being so quick about it!
I did not know I was actually submitting a feature request, actually thought I was missing something.
Thanks for being awesome!

I hope I do not overextend my stay, but may I ask, as I am new to much of the github stuff, when or how the npm module gets updated? I can surely find a way to hack the commit into the currently installed node module, but I'd rather do it cleanly. (Also for possible later use by others)
I found that the last npm update was 5 month ago.
